src/components/atoms/Button/Button.styles.ts

```diff
@@ -9,7 +9,7 @@ export const Button = styled.button<ButtonProps>`
 
   border-radius: 1.375em;
   border: none;
-  background-color: #361f6e;
+  background-color: #6f43ff;
```

Comment:
use our privary color here, but I can't import it from `constants` because it's styled components